CHALLENGE 1 
--> download dependencies, not compatible with python 3.9
--> had to create a virtual environment

--> get the subdirectory correctly
.DS_Store
--> understand how cv works

-->can't use variables?
 ValueError: Shapes (None, 3) and (None, 43) are incompatible

--> low accuracy
333/333 - 1s - loss: 3.5034 - accuracy: 0.0530 - 1s/epoch - 4ms/step